pycqed/measurement/measurement_control.py

```diff
@@ -1128,6 +1127,13 @@ def save_instrument_settings(self, data_object=None, *args):
         parameter. Only saves the value and not the update time (which is
         known in the snapshot)
         '''
+
+
+        import numpy
+        import sys
+        opt = numpy.get_printoptions()
+        numpy.set_printoptions(threshold=sys.maxsize)
```

Comment:
Does this avoid `'[1,2,...,100]'` strings?

Reply:
yes

Reply:
:+1: